test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -14711,11 +14731,22 @@ def run(self):
                 _p.expect(out)
             self.logger.info('sending exit')
             _p.sendline("exit")
-            self.logger.info('waiting for the subprocess to finish')
-            _p.wait()
-            _p.close()
-            self.job.interactive_handle = None
-            self.logger.debug(_p.exitstatus)
+
+            if is_local:
```

Comment:
Not sure but do we really need this if else? I think we can use else part for both local/remote

Reply:
I am not sure if we can combine both of them into one block. I will give it a try.